### ISSUE TITLE
[pre alpha]  ERROR: Failed building wheel for faiss-cpu

### ISSUE THREAD

**Issue by msmygit** [langflow-ai/langflow#2318]
**Describe the bug**
an error causing install failures. See below for details, 

**Browser and Version**
 - Browser [e.g. chrome, safari]- n/a
 - Version [e.g. 22]- n/a
 - Python Version of the virtual environment is,
 ```
% python -V
Python 3.10.14
```

**To Reproduce**

- Run `python -m pip install langflow --pre --force-reinstall -U`, and errors as below:
```
...
Collecting grpcio-status<2.0.dev0,>=1.33.2
  Using cached grpcio_status-1.64.1-py3-none-any.whl (14 kB)
  Using cached grpcio_status-1.64.0-py3-none-any.whl (14 kB)
  Using cached grpcio_status-1.64.0rc1-py3-none-any.whl (14 kB)
  Using cached grpcio_status-1.63.0-py3-none-any.whl (14 kB)
  Using cached grpcio_status-1.63.0rc2-py3-none-any.whl (14 kB)
  Using cached grpcio_status-1.63.0rc1-py3-none-any.whl (14 kB)
Building wheels for collected packages: faiss-cpu
  Building wheel for faiss-cpu (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for faiss-cpu (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [8 lines of output]
      running bdist_wheel
      running build
      running build_py
      running build_ext
      building 'faiss._swigfaiss' extension
      swigging faiss/faiss/python/swigfaiss.i to faiss/faiss/python/swigfaiss_wrap.cpp
      swig -python -c++ -Doverride= -doxygen -Ifaiss -I/private/var/folders/q3/s2px_6w50ks2cxmcc00k3wyh0000gp/T/pip-build-env-j4tnsdqd/overlay/lib/python3.10/site-packages/numpy/core/include -Ifaiss -I/usr/local/include -o faiss/faiss/python/swigfaiss_wrap.cpp faiss/faiss/python/swigfaiss.i
      error: command 'swig' failed: No such file or directory
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for faiss-cpu
Failed to build faiss-cpu
ERROR: Could not build wheels for faiss-cpu, which is required to install pyproject.toml-based projects

[notice] A new release of pip is available: 23.0.1 -> 24.1
[notice] To update, run: pip install --upgrade pip
```

**Screenshots**
n/a

**Additional context**
n/a

**Comment by msmygit** [comment]
This is the same error even if I remove the `-pre` from the command - `python -m pip install langflow --force-reinstall -U` cc/ @rodrigosnader

**Comment by mihit** [comment]
@msmygit You don't need --force-reinstall if you want to just use langflow.

**Comment by msmygit** [comment]
okay. This is what fixed my issue of upgrading from the `1.0.0a61` --> `1.0.5`,

```
python -m pip install langflow -U
```

<details>
<summary>Expand/Collapse</summary>
    
    ```
    ...
    Installing collected packages: smmap, gitdb, gitpython, langchain-mongodb, langchain-aws, langflow-base, langflow
      Attempting uninstall: langflow-base
        Found existing installation: langflow-base 0.0.72
        Uninstalling langflow-base-0.0.72:
          Successfully uninstalled langflow-base-0.0.72
      Attempting uninstall: langflow
        Found existing installation: langflow 1.0.0a61
        Uninstalling langflow-1.0.0a61:
          Successfully uninstalled langflow-1.0.0a61
    Successfully installed gitdb-4.0.11 gitpython-3.1.43 langchain-aws-0.1.8 langchain-mongodb-0.1.6 langflow-1.0.5 langflow-base-0.0.81 smmap-5.0.1

    [notice] A new release of pip is available: 23.0.1 -> 24.1
    [notice] To update, run: pip install --upgrade pip
    ```
</details>

**Comment by msmygit** [comment]
Although, it screwed up my existing projects and wiped out everything 🤯 😭
